Appraisals

```diff
@@ -17,3 +17,13 @@ appraise 'ar-5.1_pt-5.0' do
   gem 'activerecord', '~> 5.1.0'
   gem 'paper_trail', '~> 5.0'
 end
+
+appraise 'ar-5.1_pt-10.3' do
+  gem 'activerecord', '~> 5.1'
```

Comment:
```suggestion
  gem 'activerecord', '~> 5.1.0'
```

Reply:
This would constrain it to `5.1.0.x`, right? Why wouldn't we want to allow it to go with the latest patch level, e.g. `5.1.7` that we're going to need to be able to run it against? Haven't done a ton of dependency work like this for a while so forgive me if I'm misunderstanding or missing something.

Reply:
`~> 5.1.0` means `5.1.x` see https://thoughtbot.com/blog/rubys-pessimistic-operator and https://guides.rubygems.org/patterns/#pessimistic-version-constraint

Reply:
Sigh. I even read the rubygems.org page before asking that question :sob: